src/Core/Transform/MutualInformationAffine/MutualInformationAffine.ipynb

```diff
@@ -0,0 +1,1973 @@
+{
```

Comment:
<p>Use Pythonic version as commented in the other notebook.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKExamples/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>